### ISSUE TITLE
Rethinking the feature of  the centerness layer

### ISSUE THREAD

**Issue by Wufang1997** [tianzhi0549/FCOS#344]
Hi, FCOS is really a delicate model. I noticed that you improved the performance by sharing features of the classification layer with the centerness layer. However, I am just curious about that whether you have tried to separate the centerness layer with both the classification layer and the regression layer? In other words, the centerness is directly using its own feature processing layer.

**Comment by tianzhi0549** [comment]
@Wufang1997 Yes. We tried this, but it only results in minor improvements on COCO.
